src/components/MicroblocksSequence.vue

```diff
@@ -7,7 +7,7 @@
       tag="div">
       <div
         v-for="microblock in microblocks"
-        :key="microblock.hash"
+        :key="microblock.hash + microblock.prev_key_hash"
```

Comment:
What is this needed for? I would expect some explanation here as it is out of the scope of initial issue.

Reply:
You are right, this change deserves some extra explanation:

Unrelated fix for microblocks with the same hash that are created twice with the same hash due to microfork happening on new keyblock mint